test/test_markov_approx.jl

```diff
@@ -16,4 +16,27 @@
             @test isapprox(sum(mc.state_values), 0.0; rough_kwargs...)
         end
     end
+
+
+    #
+    # Test discrete estimation
+    #
+    P = [0.5 0.25 0.25
+         0.25 0.5 0.25
+         0.25 0.25 0.5]
+
+    # Test with numerical inputs
+    mc = MarkovChain(P, [0.0, 0.5, 1.0])
+    X = simulate(mc, 100_000)
+    mc2 = estimate_MC_discrete(X)
+    @test isapprox(mc.state_values, mc2.state_values, atol=1e-10)
+    @test isapprox(mc.p, mc2.p, atol=1e-2)
+
+    # Test with other inputs
+    mc = MarkovChain(P, ["a", "b", "c"])
+    X = simulate(mc, 100_000)
+    mc2 = estimate_MC_discrete(X)
+    @test isapprox(mc.state_values, mc2.state_values, atol=1e-10)
+    @test isapprox(mc.p, mc2.p, atol=1e-2)
```

Comment:
A test where only a subset of states appears in the simulated path would be great.

Something like 


```
P = [1 0 0; 0.5 0.5 0; 1/3 1/3 1/3]
mc = MarkovChain(P, [10, 20, 30])
X = simulate(mc, 100_000, init=1)
```

Then the result should only be 2x2 and have values [10, 20]